diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+_Graces-MacBook-Pro:~ grace$~/wdi/try$_
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+_GitHub is a website that allows us to save code and share code with others throughtout the internet.  In addition, allows us to work on issues while making copies of your work so if the code you worked on is not the issue, you can revert back to the original and work on a different part._
```

Comment:
It is a website but more simple, it is 'a user interface (UI) for accessing git's version control capabilities'. Git allows you to revert back to different versions of projects.